__tests__/app.test.js

```diff
@@ -20,4 +21,30 @@ describe( "/api/topics", ()=> {
             })
         })
     })
-})
\ No newline at end of file
+    it("GET 404 when an endpoint does not exist", () => {
+        return request(app)
+        .get("/api/banana")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("path not found")
+            })
+        })
+    })
+
+
+describe( "/api/", ()=> {
+    it("GET 200 sends an object of all available endpoints", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({body}) => {
+            const valuesOfAllEndpoints = Object.values(body);
+            const allRequests = Object.keys(body);
+            valuesOfAllEndpoints.forEach((endpoint) => {
+                expect(Object.keys(endpoint)).toEqual([ 'description', 'queries', 'exampleResponse' ])
+            })
+            expect(allRequests).toEqual([ 'GET /api', 'GET /api/topics', 'GET /api/articles' ])
```

Comment:
When you add information for future endpoints, this line will fail as you have added more information to your json file. Because you are already testing that each endpoint has description/queries/exampleResponse keys, and that the entire body matches your JSON file, I would be suggest just removing this line entirely